ops/01-linux-fs.md

```diff
@@ -0,0 +1,436 @@
+# Linux 文件系统的基本概念与操作
+
+::: info 本文信息
+作者：thezzisu<sub>with Claude Sonnet</sub>
+
+状态：待审阅
+:::
+
+## 块设备、磁盘与分区的概念
+
+在开始学习 Linux 文件系统之前，我们需要先理解几个基本概念，它们之间的关系如下：
+
+```mermaid
+graph TB
+    A[物理硬件] --> B[块设备]
+    B --> C[磁盘]
+    C --> D[分区]
+    D --> E[文件系统]
+```
+
+### 块设备
+
+块设备是一种以块（block）为单位进行数据读写的设备。与字符设备（按字符读写）不同，块设备允许随机访问数据，可以跳到任意位置读写数据。最常见的块设备就是硬盘。
+
+一些关键特点：
+
+- 以固定大小的块为单位读写数据（通常是 512 字节或 4KB）
+- 支持随机访问，可以直接跳转到任意位置
+- 在 Linux 中以特殊文件形式存在，通常位于 `/dev` 目录下
+
+### 磁盘
+
+磁盘是最常见的块设备，在 Linux 系统中有不同的命名规则：
+
+- IDE/SATA 硬盘：`/dev/sda`, `/dev/sdb`, `/dev/sdc`...
+- NVMe 固态硬盘：`/dev/nvme0n1`, `/dev/nvme0n2`...
+- 虚拟磁盘：`/dev/vda`, `/dev/vdb`...
+
+其中字母表示不同的物理设备，数字表示分区编号。
+
+### 分区
+
+分区是将一个物理磁盘划分为多个逻辑区域的方式。分区的主要目的是：
+
+- 更好地组织和管理数据
+- 提高系统性能和安全性
+- 支持多个操作系统共存
+
+分区类型：
+
+1. MBR 分区表（传统）
+
+   - 最多支持 4 个主分区
+   - 或 3 个主分区 + 1 个扩展分区（扩展分区可包含多个逻辑分区）
+   - 单个分区最大支持 2TB
+
+2. GPT 分区表（现代）
+
+   - 支持最多 128 个主分区
+   - 支持超过 2TB 的分区大小
+   - 更好的容错性
+
+### 文件系统
+
+文件系统是在分区之上建立的数据组织方式，它决定了如何在分区中存储和管理文件。常见的 Linux 文件系统类型包括：
+
+- `ext4`：Linux 最常用的文件系统，功能强大且稳定
+- `xfs`：适合大容量存储，性能优秀
+- `btrfs`：新一代文件系统，支持快照等高级特性
+- `fat32/ntfs`：主要用于与 Windows 系统交互
+
+## Linux 的挂载机制
+
+Linux 系统中的所有文件都在一个统一的目录树中，通过挂载将物理设备整合到这个目录树中：
+
+```mermaid
+graph LR
+    A[物理设备] --挂载--> B[挂载点]
+    B --> C[统一目录树]
+    C --> D[/]
+    C --> E[/home]
+    C --> F[/mnt]
+```
+
+### 挂载的概念
+
+挂载（mount）是将一个文件系统关联到目录树的过程。可以把它想象成把一个新的抽屉（文件系统）安装到书柜（目录树）上。
+
+为什么需要挂载：
+
+- 保持文件系统的统一性
+- 灵活管理不同的存储设备
+- 实现文件系统的动态扩展
+
+### 挂载点与目录树的关系
+
+- 挂载点是目录树中的一个目录
+- 挂载后，访问该目录就等于访问被挂载的文件系统
+- 常用的挂载点位置：
+  - `/mnt`：临时挂载点
+  - `/media`：可移动设备自动挂载点
+  - `/`：根文件系统挂载点
+
+### 自动挂载与手动挂载
+
+Linux 系统支持两种挂载方式：
+
+1. 自动挂载
+
+   - 系统启动时通过 `/etc/fstab` 自动挂载
+   - 插入可移动设备时自动挂载
```

Comment:
This may be confusing for people who don't know the relevant knowledge. It's usually mounted by `udisks2` whether it's removable or not.